app/models/user.rb

```diff
@@ -27,14 +30,22 @@ class User < Common::RedisStore
   attribute :participant_id
   attribute :ssn
 
+  # mvi 'golden record' data
+  attribute :mvi
+
   validates :uuid, presence: true
   validates :email, presence: true
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :birth_date, presence: true
+  validates :ssn, presence: true
 
   def rating_record
     client = EVSS::CommonService.new(self)
     client.find_rating_info.body.fetch('ratingRecord', {})
   end
 
+  # TODO(AJD): realize this is temporary but it's also used in specs where it should be stubbed or a factory
```

Comment:
`self.sample_claimant` is a temporary hack, but long term we do need a way to let testers in stage/dev/local sign in as mock users. I'm not sure what the best way to handle this will be. We should talk about it at the engineering meeting today.